config/config.go

```diff
@@ -123,14 +123,16 @@ type PropertyRecvOptions struct {
 }
 
 type PushJob struct {
-	ActiveJob    `yaml:",inline"`
-	Snapshotting SnapshottingEnum  `yaml:"snapshotting"`
-	Filesystems  FilesystemsFilter `yaml:"filesystems"`
-	Send         *SendOptions      `yaml:"send,fromdefaults,optional"`
+	ActiveJob     `yaml:",inline"`
+	Snapshotting  SnapshottingEnum  `yaml:"snapshotting"`
+	Filesystems   FilesystemsFilter `yaml:"filesystems"`
+	Send          *SendOptions      `yaml:"send,fromdefaults,optional"`
+	JobSnapPrefix string            `yaml:"replication_snapshots_prefix,optional"`
```

Comment:
Needs coverage in docs.

Reply:
added some docs, I hope I did it right and also it is enough, feel free to change anything.

Reply:
I got some errors with the circleci checks.
It looks better now but the test-go-on-latest-go-release check is still failing, no idea why yet, any help is welcome:)